specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -1241,16 +1238,16 @@ unique name or allow the Deployer to choose a unique name, but must not
 fail the deployment. This ensures that existing applications continue to
 be deployable.
 . The deployment tool must open each of the
-Java EE modules listed in the Java EE application deployment descriptor
-or discovered using the rules above and read the Java EE module
+Jakarta EE modules listed in the Jakarta EE application deployment descriptor
+or discovered using the rules above and read the Jakarta EE module
 deployment descriptor, if present in the package. See the Enterprise
-JavaBeans, servlet, Java EE Connector and application client
+JakartaBeans, servlet, Jakarta EE Connector and application client
 specifications for the required location and name of the deployment
 descriptor for each component type. Deployment descriptors are optional
 for all module types. (The application client specification is
 link:#a3294[See Application Clients]”.)
 . If the module deployment descriptor is
-absent, or is present and is a Java EE 5 or later version descriptor and
+absent, or is present and is a Jakarta EE 5 or later version descriptor and
```

Comment:
again not sure whether this should say Java EE 5

Reply:
reverted